Library/Formula/abcde.rb

```diff
@@ -22,6 +23,16 @@ class Abcde < Formula
   depends_on "vorbis-tools" => :optional
   depends_on "glyr" => :optional
 
+  # Fix "Expansion of $REDIR quoted on MacOSX breaks cp" bug
+  # (http://abcde.einval.com/bugzilla/show_bug.cgi?id=22)
+  # Obsolete once a new version is released as upstream already merged this
+  # patch in c024365a846faae390f23c86f32235d6209e6edf
+  # (http://git.einval.com/cgi-bin/gitweb.cgi?p=abcde.git;a=commit;h=c024365a846faae390f23c86f32235d6209e6edf)
+  patch do
+    url "http://abcde.einval.com/bugzilla/attachment.cgi?id=12"
```

Comment:
Nope, but then it also patches the changelog file which is probably the culprit here.